m1.wack-a-mole/script.js

```diff
@@ -0,0 +1,69 @@
+$(document).ready(function(){
+       let score, timeLeft, randomTime, currentHole;
+       const holes = $('.hole');
+       const moles =  $('.mole');
+
+    moles.click(function () {
+        let currentMole = $(this);
+        if(currentMole.hasClass('showMole')){
+            score++;
+            $('.score').html(score);
+        }
+    });
+
+    $('#startBtn').on('click', function(){
+        timeLeft = 30;
+          score = 0;
+          $(this).prop("disabled", true);
+          $(this).css("cursor", "no-drop");
+          setTimer();
+          setInterval(() => {
+              if(timeLeft > 0){
+                  randomTime = getRandomTime();
+                  currentHole = convertToJqueryElem(getRandomHole(holes));
```

Comment:
since you only use the `.hole` element you can change "getRandomHole " to "getRandomMole "
which should return the only child of a  hole, which is a mole.
another option is to use `find("selector")` or [first-child](https://api.jquery.com/first-child-selector/) selector, to extract mole from a JQuery hole `div` element
you can use